### ISSUE TITLE
Support exact timestamps as arguments for moderation commands

### ISSUE THREAD

**Issue by MarkKoz** [python-discord/bot#458]
> Yesterday (for me) we briefly talked about being able to set exact dates instead of a relative duration for the expiration field. I've checked and I've not seen it used, so I'm not married to it. If we do want it, I may be better to have a separate converter that converts datetime strings of a certain format to a datetime instead of creating a hybrid one that both parses relative durations and absolute datetime strings.
> 
> I currently see not no real use for it, though, although it could be useful if we ever want to do something with absolute dates (like adding events to the event calendar we've envisioned.)

_Originally posted by @SebastiaanZ in https://github.com/python-discord/bot/pull/457_

> Exact dates could be handy for underage bans.

_Originally posted by @scragly in https://github.com/python-discord/bot/pull/457_

**Comment by MarkKoz** [comment]
The converter has been added but it currently isn't being anywhere. I think it should be done in a separate PR, after #457.
